app/models/topics.js

```diff
@@ -0,0 +1,10 @@
+const db = require('../../db/connection');
+
+exports.selectTopics = () => {
+    return db.query('SELECT * FROM topics').then((result) => {
+        if (result.rows.length === 0){
+            return Promise.reject({status: 404, msg: 'No topics exist'})
```

Comment:
No need to custom handle as per my other comments. Please delete this part of the code.